apps/cli/commands/ingest_reddit.py

```
@@ -0,0 +1,73 @@
+        console.print("[yellow]Chunking...[/yellow]")
+        all_chunks = []
+        for norm_doc in normalized_docs:
+            all_chunks.extend(chunker.chunk(norm_doc))
+        console.print(f"[green]✓ Created {len(all_chunks)} chunks[/green]")
+
+        console.print("[yellow]Generating embeddings...[/yellow]")
+        embeddings = embedder.embed_texts([chunk.text for chunk in all_chunks])
+
+        console.print("[yellow]Writing to Qdrant...[/yellow]")
+        qdrant_writer.upsert_batch(chunks=all_chunks, embeddings=embeddings)
+        console.print(f"[green]✓ Reddit ingestion complete![/green]")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)